apb.yml

```diff
@@ -1,19 +1,19 @@
 version: 1.0
-name: prometheus-apb
-description: Install prometheus
+name: metrics-apb
+description: Install metrics service based on prometheus
 bindable: False
 async: optional
 tags:
   - mobile-service
 metadata:
-  displayName: Prometheus
-  imageUrl: "https://avatars1.githubusercontent.com/u/3380462?s=200&v=4"
+  displayName: Metrics
+  imageUrl: "https://avatars2.githubusercontent.com/u/34546838?s=200&v=4"
```

Comment:
@pb82 I think it would look better to keep the prometheus logo rather than the aerogear catalog logo. The usage of the logo seems to be ok https://groups.google.com/forum/#!topic/prometheus-users/DMBfaIpIC_s

Reply:
Agree, the aerogear-catalog logo does not really look that nice in the service catalog. Will change back to the prometheus logo for now.